### PR TITLE
LPS-183084 | Test Automation | Master

### PR DESCRIPTION
**Pull request by rafaellajordao**
# LPS-183084 - Product QA | Test Automation Creation

Hi! :)
This pr is corresponding to this ticket: [LPS-182032](https://issues.liferay.com/browse/LPS-182032) → [LPS-183084](https://issues.liferay.com/browse/LPS-183084)

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by rafaellajordao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/contentdashboard/ContentDashboard.path [hunk 1]
```diff
@@ -265,6 +265,11 @@
 	<td>//*[contains(@class, 'lexicon-icon lexicon-icon-order')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>REVIEW_DATE_COLUMN_VALUE</td>
+	<td>//td[contains(@class,'lfr-review-date-column') and contains(text(),'${reviewDate}')]</td>
+	<td></td>
+</tr>
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/contentdashboard/ContentDashboard.path:272]: Use general locator
LexiconTable#TABLE_ENTRY_INFO

File: portal-web/test/functional/com/liferay/portalweb/macros/journal/WebContent.macro [hunk 2]
```diff
@@ -316,6 +316,28 @@ definition {
 			value1 = ${priority});
 	}
 
+	macro addReviewDate {
+		Panel.expandPanel(panel = "Schedule");
+
+		AssertChecked(
+			locator1 = "Checkbox#NEVER_REVIEW",
+			value1 = "Never Expire");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/journal/WebContent.macro:324]: No need

File: portal-web/test/functional/com/liferay/portalweb/macros/journal/WebContent.macro [hunk 3]
```diff
@@ -316,6 +316,28 @@ definition {
...
+
+		Uncheck(
+			locator1 = "Checkbox#NEVER_REVIEW",
+			value1 = "Never Expire");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/journal/WebContent.macro:328]: invalid parameter

File: portal-web/test/functional/com/liferay/portalweb/macros/journal/WebContent.macro [hunk 4]
```diff
@@ -316,6 +316,28 @@ definition {
...
+
+		Type.clickAtType(
+			locator1 = "TextInput#REVIEW_DATE",
+			value1 = ${reviewDate});
+
+		if (isSet(reviewTime)) {
+			Type.clickAtType(
+				locator1 = "TextInput#REVIEW_DATE_TIME",
+				value1 = ${displayTime});
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/journal/WebContent.macro:337]: reviewTime

**Review by Tim-Cao — COMMENTED** [review]
Hi @rafaellajordao, would you mind refactoring according to comments? Thanks

**Review by Tim-Cao — COMMENTED** [review]
Hi @rafaellajordao, would you mind refactoring according to comments? Thanks

**Review by Tim-Cao — COMMENTED** [review]
Hi @rafaellajordao, would you mind refactoring according to comments? Thanks

**Review by Tim-Cao — COMMENTED** [review]
Hi @rafaellajordao, would you mind refactoring according to comments? Thanks

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
See changes https://github.com/liferay-echo/liferay-portal/pull/12099
@rafaellajordao

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**